openvino/itt/CMakeLists.txt

```diff
@@ -21,7 +21,10 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 file(GLOB_RECURSE SOURCES "src/*.cpp")
 
 if(ENABLE_PROFILING_ITT)
-    find_package(ITT REQUIRED)
```

Comment:
Why do we need this? 
If user enables this flag, he expects that ITT will be enabled. If ITT wasn't found it is better to have an error, because not all users read warnings.

Reply:
Usually cmake works in the following way - try to find all the libraries / dependencies and reports the status. In our case, it fails with error if ITT is not found. So, the idea is to follow usual semantic and `ENABLE_PROFILING_ITT` is used only to explicitly disable ITT usage

Reply:
@vladislav-volkov looks like we need to enable `ENABLE_PROFILING_ITT` by default.

Reply:
ok, I'll enable it